azure-pipelines.yml

```diff
@@ -1,34 +1,102 @@
-# ASP.NET
-# Build and test ASP.NET projects.
-# Add steps that publish symbols, save build artifacts, deploy, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
-
 trigger:
```

Comment:
Maybe add a `batch: true` statement, and exclude `README` and `images/` folder from build trigger?

Reply:
noted, added